commitizen/defaults.py

```diff
@@ -98,6 +101,9 @@ class Settings(TypedDict, total=False):
     "prerelease_offset": 0,
     "encoding": encoding,
     "always_signoff": False,
+    "format": None,  # default guessed from changelog_file
```

Comment:
I think it might be better for us to change it to `changelog_format` as well

Reply:
Renamed into `changelog_format` 👍🏼